command/agent/consul/syncer.go

```diff
@@ -82,14 +84,6 @@ type Syncer struct {
 	trackedChecks   map[string]*consul.AgentCheckRegistration
 	trackedServices map[string]*consul.AgentServiceRegistration
 
-	// serviceRegPrefix is used to namespace the domain of registered
-	// Consul Services and Checks belonging to a single Syncer.  A given
-	// Nomad Agent may spawn multiple Syncer tasks between the Agent
-	// Agent and its Executors, all syncing to a single Consul Agent.
-	// The serviceRegPrefix allows multiple Syncers to coexist without
-	// each Syncer clobbering each others Services.  The Syncer namespace
-	// protocol is fmt.Sprintf("nomad-%s-%s", serviceRegPrefix, miscID).
-	// serviceRegPrefix is guarded by the registryLock.
 	serviceRegPrefix string
```

Comment:
Can you toss a comment here to describe the difference between the `serviceRegPrefix` and the `groupName` above?